File: flows/flow-1/scriptlets/+scriptlet#1.py
from oocana import Context
import pandas as pd
#region generated meta
import typing
Inputs = typing.Dict[str, typing.Any]
class Outputs(typing.TypedDict):
    output: typing.Any
#endregion

def main(params: Inputs, context: Context) -> Outputs | None:

    # your code

    df = pd.DataFrame({'A': [1, 2, 3], 'B': ['a', 'b', 'c']})
    context.preview(df) # pyright: ignore[reportArgumentType]
    return {"output": "Hello, world!"}

File: flows/flow-1/scriptlets/+scriptlet#1.ts
//#region generated meta
type Inputs = {
    output: any;
};
type Outputs = {
};
//#endregion

import type { Context } from "@oomol/types/oocana";
import isplainObject from "lodash.isplainobject"

export default async function(
    params: Inputs,
    context: Context<Inputs, Outputs>
): Promise<Partial<Outputs> | undefined | void> {

    const b = isplainObject(context);
    console.log("isplainObject:", b)
};
